### ISSUE TITLE
How many domains should a student get in touch with?

### ISSUE THREAD

**Issue by MakisH** [the-teachingRSE-project/RSE-Masters#4]
Besides the RSE skills, a student already has their home domain from their bachelor's (see #1). Should they formally get in touch with a third domain as well?

Should a Chemistry graduate studying RSE also get a course on History?

**Comment by juliandehne** [comment]
I think RSE education and one Domain would be sufficient. I am not clear why the student should change the domain in the masters? Possibly, I missed the discussion here.

**Comment by jpthiele** [comment]
I think a student should be allowed to, if the number of courses permits, but I would not make it mandatory.
For example my computational engineering curricula (B.Sc. and M.Sc.) had  a 30 ECTS "Wahlpflicht" module for engineering with the constraint that 15 ECTS should be done in one of the fields, i.e. mechanical, electrical and civil eng.
I could envision something similar for the domain courses but it highly depends on the amount of ECTS we have available for it.

**Comment by CaptainSifff** [comment]
My opinion, was ,that if people want to go down to a specialized $DOMAIN-RSE, which probably requires continuing some of the $DOMAIN Master courses, then a single intro into another domain is sufficient. If they aim for a career in e.g. an RSE-unit( @juliandehne  you probably haven't seen this: https://github.com/DE-RSE/2023_paper-RSE-groups  ), 
where they might cater to multiple domains, they should probably aim for two "different" fields from their home domain.
How far away? If you have done mechanical engineering before, then e.g. electrical engineering and biology would be OK.

**Comment by juliandehne** [comment]
I agree with both perspectives. I think the question is what the study program is supposed to qualify for. University has the issue that it targets mainly science and other jobs are sort second plan. In that case, @CaptainSifff argument that domain science should be a valid profile holds. The well-rounded RSE generalist might be more like a specialization of a CS program. However, if it "only" contains the general skill set, a one-year master might be sufficient and definiately does not need a domain science at all. Maybe we can use CS-RSE and MNT-RSE as references for the two main ideas/profiles for the future discussion?
